test/functorch/test_control_flow.py

```diff
@@ -1744,6 +1761,102 @@ def fct_different_output_tuple(x, y):
         )
         self.assertEqual(result_diff, expected_result)
         self.assertEqual(result_diff[1], result_same[1][1])
+        
+    @unittest.skipIf(not SM70OrLater, "triton")
+    @requires_cuda
+    @parametrize("compile_mode", ["none", "compile", "compile_dynamic_shape"])
+    @parametrize("combine_mode", ["pointwise", "generic"])
+    @parametrize("reverse", [False, True])
+    @parametrize("device", [torch.device("cpu"), torch.device("cuda")])
+    # Skipping combine_mode=pointwise
+    # as the expand operation is non-pointwise
+    @decorateIf(
```

Comment:
We probably shouldn't parametrize on "combine_mode" since we're gonna skip it.

Reply:
I went through the testcases and tried to remove the unnecessary parametrizations